src/test/java/com/thebois/models/beings/PawnTests.java

```diff
@@ -0,0 +1,48 @@
+package com.thebois.models.beings;
+
+import org.junit.jupiter.api.Test;
+
+import com.thebois.models.Position;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class PawnTests {
+
+    @Test
+    public void arriveAtDestinationTest() {
+        // Arrange
+        final Position startPosition = new Position(0, 0);
+        final Position endPosition = new Position(3, 4);
+
+        final Pawn pawn = new Pawn(startPosition, endPosition);
+
+        // Act
+        final int steps = 5;
+        for (int i = 0; i < steps; i++) {
+            pawn.update();
+            System.out.println(pawn.getPosition().getPosX() + " " + pawn.getPosition().getPosY());
```

Comment:
I think we should avoid having prints in our tests, and overall in our code. Guessing that its left from some debugging?